Apps/W1/Shopify/app/src/Customers/Codeunits/ShpfyCustomerExport.Codeunit.al

```diff
@@ -87,9 +88,12 @@ codeunit 30116 "Shpfy Customer Export"
     var
         ShopifyCustomer: Record "Shpfy Customer";
         CustomerAddress: Record "Shpfy Customer Address";
+        EmptyEmailAddressLbl: Label 'Customer has no e-mail address.';
     begin
-        if Customer."E-Mail" = '' then
+        if Customer."E-Mail" = '' then begin
```

Comment:
We have the exact logic here: https://github.com/microsoft/ALAppExtensions/blob/main/Apps/W1/Shopify/app/src/Companies/Codeunits/ShpfyCompanyExport.Codeunit.al

We should probably add it there too